pkg/scheduler/scheduling_partition.go

```diff
@@ -356,6 +356,17 @@ func (psc *partitionSchedulingContext) removeSchedulingNode(nodeID string) {
 	}
 }
 
+func (psc *partitionSchedulingContext) calculateOutstandingRequests() *outstandingRequests {
```

Comment:
Suggest to rename to `getAllocatableOutstandingRequests`

Reply:
see above, not sure that this is allocatable